benchmarks/grain_size_pinned_state/grain_size_plunge.ipynb

```diff
@@ -0,0 +1,467 @@
+          ]
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "7.63658755912162e-07"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 6
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Our grain size (7.64e-5 m) is similar, but slightly smaller than plotted in Figure 4 of the paper.\n",
+        "\n",
+        "In addition, we can also compare the viscosity (plotted in Figure 4, right panel):"
+      ],
+      "metadata": {
+        "id": "VhDvzRp_janV"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "D7ou_Eq60xqm"
+      },
+      "source": [
+        "In ASPECT, we compute the viscosity in the following way:\n",
+        "\n",
+        "\\begin{equation}\n",
+        "\\eta = \\frac{1}{A^{1/n}} R^m \\dot\\varepsilon_{II}^\\frac{1-n}{n} \\exp\\left({\\frac{E_d}{nR_gT}}\\right)\n",
+        "\\end{equation}\n",
+        "\n",
+        "Note that there is a difference compared to the definition in the Mulyukova paper, which uses\n",
+        "\n",
+        "\\begin{equation}\n",
+        "\\eta_\\text{disl} = \\frac{1}{2} \\frac{1}{A \\tau^{n-1}}  = \\frac{1}{2} \\frac{1}{A_0} \\tau^{1-n} \\exp\\left({\\frac{E_\\text{disl}}{R_gT}}\\right)\\\\\n",
+        "\\eta_\\text{diff} = \\frac{1}{2} \\frac{r^m}{B}  = \\frac{1}{2} \\frac{1}{B_0} R^m \\sqrt{h_G}^{m} \\exp\\left({\\frac{E_\\text{diff}}{R_gT}}\\right)\n",
+        "\\end{equation}\n",
+        "\n",
+        "They use the roughness rather than the grain size in their equation.\n",
+        "So we have to either scale our prefactor B for diffusion creep by $\\sqrt{h_G}^m$, or convert from grain size to roughness first."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "dislocation_viscosity = 1/(2 * A * stress ** (stress_exponent - 1))\n",
+        "diffusion_viscosity   = (grain_size*2/np.pi) ** (grain_size_exponent)/(2 * B)\n",
+        "\n",
+        "viscosity             = diffusion_viscosity * dislocation_viscosity / (diffusion_viscosity + dislocation_viscosity)\n",
+        "\n",
+        "print(viscosity)"
+      ],
+      "metadata": {
+        "id": "kwGM8MvEKtRj",
+        "outputId": "cbbb1b90-a6ba-4c6f-a864-1007b34703fb",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        }
+      },
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "2.8867405314847165e+18\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Again, our viscosity (2.89e18 Pa s) is similar but slightly smaller than what is plotted in the figure (approximately 3.4e18 Pa s). This is expected, since we are in the diffusion creep regime, where a lower grain sizes reduces the viscosity."
```

Comment:
```suggestion
        "Again, our viscosity (2.89e18 Pa s) is similar but slightly smaller than what is plotted in the figure (approximately 3.4e18 Pa s). This is expected, since we are in the diffusion creep regime, where a lower grain size reduces the viscosity."
```